### PR TITLE
amap: disable

### PR DESCRIPTION
**Pull request by gromgit**
Rationale: https://github.com/Homebrew/homebrew-core/pull/65291#issuecomment-750797020

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/amap.rb [hunk 1]
```diff
@@ -20,6 +20,8 @@ class Amap < Formula
     sha256 "1361e89caf2590146c5872907f90ad67ac9b99d2198320691e9f6df0cfdbe16c" => :sierra
   end
 
+  disable! date: "2018-06-04" because: :unmaintained
```
Comment (mitchblank) [inline Formula/amap.rb:23]: missing a comma

Comment (gromgit) [reply Formula/amap.rb:23]: Argh, will fix.

Sidenote: Had to submit this without my usual check, because `brew audit` just bails on disabled formulae.

Comment (carlocab) [reply Formula/amap.rb]: Note that choosing this disable date is basically equivalent to just removing the formula now. I'd choose a date closer to today.

Comment (SMillerDev) [reply Formula/amap.rb]: Okay, let's do the release date of Big Sur

**Pull request merged**